packages/polaris-viz/src/components/ChartSkeleton/ChartSkeleton.tsx

```diff
@@ -90,6 +127,8 @@ export function ChartSkeleton({
             }}
             state={state}
             errorText={errorText}
+            showLegend={props.showLegend ?? true}
+            size={props.size ?? 'small'}
```

Comment:
Thanks! Glad to hear that.

Unfortunately TS complains when I destructure them with the rest of the props. I get the message: `Property 'showLegend' does not exist on type 'Props'.ts(2339)`. I believe it's because these two props only exist on the `SimpleNormalizedSkeletonProps` type, and TS doesn't know what subtype its receiving until we get to the `case` blocks in the switch statement.

I can destructure them after line 121 though to make it cleaner.